src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -457,4 +457,19 @@
   <data name="MissingKeepAlive" xml:space="preserve">
     <value>Missing argument for '/keepalive' option.</value>
   </data>
+  <data name="AnalyzerTotalExecutionTime" xml:space="preserve">
+    <value>Total analyzer execution time: {0} seconds.</value>
+  </data>
+  <data name="MultithreadedAnalyzerExecutionNote" xml:space="preserve">
+    <value>NOTE: The actual wall clock time is likely lesser than the above execution time due to multithreaded analyzer execution.</value>
```

Comment:
:+1: I like that better too, somehow I haven't been getting github notifications for comments today so missed this one. Thanks for pointing it out.